tests/time_tests/src/timetest_infer.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <inference_engine.hpp>
+#include <iostream>
+
+#include "time-testhelper/timer.h"
```

Comment:
@asomsiko will you update `time-testhelper` name?

Reply:
Will need separate PR for this